Cortana with Outlook desktop is supported (#384)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/misc/cortana-integration.md

There are a few things to be aware of before you start using Cortana in Office 365. If you run into issues, check out [these troubleshooting tips](https://go.microsoft.com/fwlink/?LinkId=620763).
  

> - To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. You can also use Cortana with the Outlook desktop app.

Reviewer: ```suggestion
- You can add your Outlook or Office 365 account as a connected service to Cortana and add reminders and tasks to your lists in Microsoft To-Do. You can also use Cortana to read and search messages in the Outlook desktop app.
```

> - To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. (You can also use Cortana with Outlook on the desktop.)

Reviewer 2: Copy checked - no changes.

> - To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. (You can also use Cortana with Outlook on the desktop.)

Reviewer 3: I Suggest:
You can also use Cortana with **the Outlook desktop app.**
I suggest losing the (  ) too.

> - To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. (You can also use Cortana with Outlook on the desktop.)

Reviewer: ```suggestion
- To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. You can also use Cortana with the Outlook desktop app.
```

> - To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. (You can also use Cortana with Outlook on the desktop.)

Reviewer: ```suggestion
- To use Cortana in Office 365 scenarios, you must set up the Windows 10 Outlook Universal Mail and Calendar apps with Office 365. You can also use Cortana with the Outlook desktop app.
```
